meta-resin-common/recipes-core/packagegroups/packagegroup-resin.bb

```diff
@@ -15,6 +15,7 @@ include packagegroup-resin.inc
 RDEPENDS_${PN} += " \
     docker-disk \
     dosfstools \
+    healthdog \
```

Comment:
I would expect that the services using this healthdog to depend on it. For example if openvpn service will start using it, it will RDEPEND on healthdog.

Reply:
Yeah this commit is just a placeholder to force the build to happen, it should go away before merging.